src/web/lib/hakuneko/frontend@classic-dark/chapters.html

```diff
@@ -436,10 +437,16 @@
                     return;
                 }
                 this.set( 'selectedChapter', e.model.item );
+                Engine.ChaptermarkManager.addChaptermark( this.selectedChapter );
```

Comment:
this line still "needed" though. its used to when preview is clicked  the chapter mark is updated.
So for example:
 when opening Tokyo ghoul and having no  chaptermark, and starting at chapter 100
the marker will  be  set there.

on the other hand i could see confusion happening with this if a user desides to go back a chapter the marker also gets updated.

Reply:
Does not seem to work on OSX.

will first convert other changes before testing on other platform again